tests/core.py

```diff
@@ -2269,7 +2269,8 @@ def test_send_smtp(self, mock_send_mime):
         self.assertEqual(['to'], call_args[1])
         msg = call_args[2]
         self.assertEqual('subject', msg['Subject'])
-        self.assertEqual(configuration.conf.get('smtp', 'SMTP_MAIL_FROM'), msg['From'])
+        self.assertEqual(configuration.conf.get('smtp', 'SMTP_MIME_FROM') or
+                         configuration.conf.get('smtp', 'SMTP_MAIL_FROM'), msg['From'])
```

Comment:
These tests need extending to check that when mime_from is set that it is used, and when it isn't set that mail_from is used.

The `conf_var` (`conf_vars1?) decorator in use in other tests is the way to set a config for the scope of a test or block.